app/scripts/components/common/banner/index.tsx

```diff
@@ -1,78 +1,141 @@
+    ...defaultGuidance.left,
+    ...leftGuidance
+  };
+
+  const rightContent = {
+    ...defaultGuidance.right,
+    ...rightGuidance
+  };
 
   return (
-    <div>
-      {isOpen && (
-        <div className='position-relative'>
-          <USWDSBanner
-            aria-label={appTitle}
-            className={type !== infoTypeFlag ? 'bg-secondary-lighter' : ''}
-          >
-            <a href={url} target='_blank' rel='noreferrer'>
-              <USWDSBannerContent
-                className='padding-top-1 padding-bottom-1'
-                isOpen={true}
-              >
-                <div dangerouslySetInnerHTML={{ __html: text }} />
+    <USWDSBanner
+      aria-label={ariaLabel ?? defaultHeaderText}
+      className={className}
+    >
+      <USWDSBannerHeader
+        isOpen={isOpen}
+        flagImg={
+          <USWDSBannerFlag src='/img/us_flag_small.png' alt={flagImgAlt} />
+        }
+        headerText={headerText ?? defaultHeaderText}
+        headerActionText={headerActionText}
+      >
+        <USWDSBannerButton
+          isOpen={isOpen}
+          onClick={() => setIsOpen((prev) => !prev)}
+          aria-controls={contentId}
+        >
+          {headerActionText}
+        </USWDSBannerButton>
+      </USWDSBannerHeader>
+
+      <USWDSBannerContent id={contentId} isOpen={isOpen}>
+        <div className='grid-row grid-gap-lg'>
+          <USWDSBannerGuidance className='tablet:grid-col-6'>
+            <USWDSBannerIcon
+              src={leftContent.icon}
+              alt={leftContent.iconAlt || ''}
+            />
+            <USWDSMediaBlockBody>
+              <p>
+                <strong>{leftContent.title}</strong>
+                <br />
+                <span
+                  dangerouslySetInnerHTML={{
+                    __html: decode(leftContent.text)
+                  }}
+                />
+              </p>
+            </USWDSMediaBlockBody>
+          </USWDSBannerGuidance>
 
-              </USWDSBannerContent>
-            </a>
-          </USWDSBanner>
-          <div className='position-absolute top-0 right-0 margin-right-3 height-full display-flex'>
-            <button
-              className='usa-button usa-button--unstyled'
-              type='button'
-              aria-label='Close Banner'
-              onClick={onClose}
-            >
-              <Icon.Close />
-            </button>
-          </div>
+          <USWDSBannerGuidance className='tablet:grid-col-6'>
+            <USWDSBannerIcon
+              src={rightContent.icon}
+              alt={rightContent.iconAlt || ''}
+            />
+            <USWDSMediaBlockBody>
+              <p>
+                <strong>{rightContent.title}</strong>
+                <br />
+                <span
+                  dangerouslySetInnerHTML={{
+                    __html: decode(rightContent.text)
+                  }}
+                />
+              </p>
+            </USWDSMediaBlockBody>
+          </USWDSBannerGuidance>
```

Comment:
⛏️ : Since this JSX block repeats twice you could extract it into a fn like `displayGuidanceBlock` and have it take in `(content: GuidanceContent)` as a param